### DISCUSSION TITLE
transaction failing continuously due to very high gas price

### DISCUSSION THREAD

**Discussion by serv-er** [Cyfrin/foundry-full-course-cu#2902] [Q&A] [answered]
`Estimated gas price: 2006.890787908 gwei

Estimated total gas used for script: 444498

Estimated amount required: 0.892058941443530184 ETH

==========================

Transactions saved to: /home/sarvesh9044/Learn-Foundary-simple-stotage/broadcast/DeploysimpleStorage.s.sol/11155111/run-latest.json

Sensitive values saved to: /home/sarvesh9044/Learn-Foundary-simple-stotage/cache/DeploysimpleStorage.s.sol/11155111/run-latest.json

Error: Failed to send transaction

Context:
- server returned an error response: error code -32000: insufficient funds for gas * price + value: balance 43476910557833283, tx cost 892058941443530184, overshot 848582030885696901`

this error continously come on  sendig transaction on sepolia testnet in metamask and each 
time gas price is auomatically increasing

**Comment by LordAdhaar** [comment]
It is possible this could be due to high network congestion on the sepolia network, you can try later and check if the gas price has reduced. 

You can check the gas price using 

`cast gas-price --rpc-url $SEPOLIA_RPC_URL`

**Reply by serv-er** [reply]
actually as you said I checked the gas price but I got only 18.031277773 Gwei

**Reply by serv-er** [reply]
but on sending transaction it is high as much as I wrote in the question also more than that when I send again a transaction

**Reply by EngrPips** [reply]
I think `18.031277773 Gwei` represents what a unit of gas costs so you will do `18.031277773 Gwei` * `the quantity of gas your transaction cost`

**Reply by serv-er** [reply]
actually if you see my question then it is written there that in the error that gas price is 2000 gwei approx but on checking by command it is only 18 gwei 
I don't know why it is demanding that much gas price on sending transaction I just stuck

**Reply by EngrPips** [reply, marked as answer]
Well, the time you sent the transaction differs from when you did the check, and the gas price fluctuates irregularly, so that makes sense. You can maybe recheck the gas price and then retry sending the transaction and see if there is a correlation in the gas price

**Reply by serv-er** [reply]
hey @EngrPips actually you were right but now there rose a new problem that the amount of transaction cost is lesser than that I have in my metamask wallet but still I'm getting same error continuously

**Reply by serv-er** [reply]
hey bro thanks for your replies and I got output as expected thanks once again

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
